nilearn/decoding/space_net.py

```diff
@@ -457,6 +459,7 @@ def path_scores(solver, X, y, mask, alphas, l1_ratios, train, test,
 
 
 class BaseSpaceNet(LinearModel, RegressorMixin, CacheMixin):
+
     """
     Regression and classification learners with sparsity and spatial priors
```

Comment:
You may as well fix the docstring while you are at it: Regression should be on the same line as the first `"""`